indico/modules/events/registration/client/js/form_setup/ItemSettingsModal.jsx

```diff
@@ -145,6 +131,7 @@ export default function ItemSettingsModal({id, sectionId, onClose}) {
 ItemSettingsModal.propTypes = {
   id: PropTypes.number,
   sectionId: PropTypes.number,
+  itemType: PropTypes.string.isRequired,
```

Comment:
I don't think this should be required; the same modal is used to edit an existing item